src/nvme/types.h

```diff
@@ -4856,6 +4856,34 @@ enum nvmf_tcp_sectype {
 	NVMF_TCP_SECTYPE_TLS13	= 2,
 };
 
+/**
+ * enum nvmf_get_log_discovery_lid_support - Discovery log specific support
+ * @NVMF_LOG_DISC_LID_NONE:	None
+ * @NVMF_LOG_DISC_LID_EXTDLPES:	Extended Discovery Log Page Entries Supported
+ * @NVMF_LOG_DISC_LID_PLEOS:	Port Local Entries Only Supported
+ * @NVMF_LOG_DISC_LID_ALLSUBES:	All NVM Subsystem Entries Supported
+ */
+enum nvmf_get_log_discovery_lid_support {
```

Comment:
The '_get_'  part should be dropped to match with the rest. Also the spec doesn't mention anything about 'get' in the `5.16.1.23 Discovery Log Page (Log Identifier 70h)` section